kubernetes/loculus/values.yaml

```diff
@@ -1519,7 +1521,6 @@ secrets:
     data:
       username: "dummy"
       password: "dummy"
-enableCrossRefCredentials: true
```

Comment:
(Using this line to create a thread on a general theme)

There are valid use cases for SeqSets without the ability to generate DOIs (i.e. without CrossRef). I'm sure those weren't supported in a super nice way before this (e.g. the DOI button was always shown even if it didn't work) but it seems like this PR probably makes that harder? I think we should maintain the possibility to run SeqSets without  specifying CrossRef credentials (if nothing else, for testing) [that may already be maintained, I'm not clear].

Reply:
Indeed, I didn't have that in mind. Should I change it such that one can have SeqSets without CrossRef?

Reply:
IMO we shouldn't change it to make it _harder_ to do that unless we have a clear reason to. I don't see one atm. I don't think we have a reason atm to invest in making SeqSets work better without crossref than they do atm (e.g. disabling the generate DOI button) but we could do.

Reply:
(we can probably still remove the `enableCrossRef credentials` thing and make it directly look for the specification of the secret in the yaml, which I think I did for something else)

Reply:
Now it's possible to set `seqSets.crossRef = null`: Then you have SeqSets, but no CrossRef integration. Locally, it worked for me.

Reply:
Thanks!